tests/foreman/cli/test_host.py

```diff
@@ -1934,12 +1936,12 @@ def test_positive_attach(module_host_subscription, host_subscription_client):
         }
     )
     result = module_host_subscription._client_enable_repo()
-    assert result.return_code == 0
+    assert result.status == 0
     # ensure that katello agent can be installed
     try:
         module_host_subscription.client.install_katello_agent()
-    except VirtualMachineError:
-        pytest.fail("VirtualMachineError raised unexpectedly!")
+    except ContentHostError:
+        pytest.fail("ContentHostError raised unexpectedly!")
```

Comment:
Please switch these to single-quote strings.

Reply:
sure, as always I forget. thanks